src/utils/alertServiceHelper.ts

```diff
@@ -3,12 +3,10 @@ export const uidGenerator = () => {
   const uuid = 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, c => {
     const r = (dt + Math.random() * 16) % 16 | 0;
     dt = Math.floor(dt / 16);
```

Comment:
do we really need such logic just to generate id's, can't we use a global counter?

Reply:
@satyamyadav @innosatyam